examples/test_all_node_examples.sh

```
@@ -0,0 +1,39 @@
+echo "Test 1: node-main (JavaScript engine)"
+echo "=========================================="
+cd node-main
+pnpm start 2>&1 | head -30
+echo
+cd ..
+
+# Test node-lite
+echo "=========================================="
+echo "Test 2: node-lite (WASM sync)"
+echo "=========================================="
+cd node-lite
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)